src/protosym/core/exceptions.py

```diff
@@ -0,0 +1,11 @@
+"""Module for all protosym exceptions."""
+
+
+class ProtoSymError(Exception):
+    """Superclass for all protosym exceptions."""
```

Comment:
```suggestion
class ProtoSymError(Exception):
    """Superclass for all protosym exceptions."""
    
    pass
```

Reply:
Done.

The blank line between the docstring and the `pass` was enforced by Black I think. Seems a bit weird to me but I guess I got to learn to live with these things. At least it is done automatically so I don't have to think about it.

Reply:
Yeah, this blank line is a Black thing.